packages/shorebird_cli/lib/src/commands/patch/ios_patcher.dart

```diff
@@ -210,12 +212,21 @@ class IosPatcher extends Patcher {
     }
 
     final patchFileSize = patchFile.statSync().size;
+    final privateKeyFile = argResults.privateKeyFile;
+    final hash = sha256.convert(patchBuildFile.readAsBytesSync()).toString();
+    final hashSignature = privateKeyFile != null
```

Comment:
Need to think more on how to share this code since it is pretty much the same as the android patcher